docs/migration_guide.rst

```diff
@@ -1,6 +1,16 @@
 Migration guide
 ===============
 
+******************************
+0.21.0 Migration guide
+******************************
+
+
+0.21.0 Minor changes
+-----------------------
+
+1. :class:`starknet_py.net.client_models.SierraContractClass` has an additional propery ``parsed_abi``
```

Comment:
This should be added below, there is already the section for `0.21.0`. Before the release we only need to remove "alpha" from the header.